src/test/scala/it/unibo/authsim/library/dsl/attack/builders/DictionaryAttackBuilderTest.scala

```diff
@@ -14,54 +17,62 @@ class DictionaryAttackBuilderTest extends AnyWordSpec:
 
     def getStatistics: Statistics = this.statistics
 
+
+  //TODO Alex appena abbiamo le policy degli algoritmi togli sta roba, te l'ho messa se no non compilava più nulla
+    @Mock
+    abstract class AlgorithmPolicy extends Policy
+    @Mock
+    val algorithmPolicy = mock[AlgorithmPolicy]
+
```

Comment:
As in the other test class, I don't understand why this code is inside the consumer class.

Reply:
Ok, I try to fix this as you suggest in the previous file